docs/source/tutorials/qaoa_quantum_dropout.ipynb

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        m_out, m_prob = c.sample()
</pre><p>why try this one-sample line, what is the meaning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>